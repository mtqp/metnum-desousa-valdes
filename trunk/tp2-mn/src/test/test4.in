1 -1 0 tests/test4.txt 0.00001