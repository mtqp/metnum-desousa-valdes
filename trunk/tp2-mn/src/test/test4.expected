0.000000
0.967054
0.000000
0.180008
0.180008
0.000000
0.000000
0.383092
0.000000
0.383092
0.454401
0.454401
0.383092
0.383092
